cypress/e2e/yves/ssp-service/ssp-service.cy.ts

```diff
@@ -205,6 +205,38 @@ interface DynamicFixtures {
         // Verify service was rescheduled
         sspServiceListPage.verifyServiceRescheduled(tomorrow);
       });
+
+      it('should allow cancelling a service', (): void => {
+        // Setup: Purchase a service as a regular customer
+        purchaseServiceAsCustomer(dynamicFixtures.customer.email, dynamicFixtures.address1.id_customer_address);
+
+        // Verify service is listed
+        sspServiceListPage.getTableRows().should('have.length.at.least', 1);
+
+        // Cancel the service
+        sspServiceListPage.cancelService();
+
+        // Verify service was cancelled and state is updated
+        sspServiceListPage.verifyServiceCancelled();
+      });
+
+      it('should not display cancel button for non-cancellable services', (): void => {
```

Comment:
we can merge these 2 tests into one. We cancel & check that there is no Cancel button anymore